src/plugins/DragDrop/index.js

```diff
@@ -2,7 +2,7 @@ const Plugin = require('../../core/Plugin')
 const Translator = require('../../core/Translator')
 const { toArray } = require('../../core/Utils')
 const dragDrop = require('drag-drop')
-const html = require('yo-yo')
+const preact = require('preact') // eslint-disable-line
```

Comment:
can we tell eslint about what JSX is being compiled down to somehow? `/** @jsx preact.h */` at the top of the file would work but maybe there's an eslint rule option that we can use?

Reply:
Added https://github.com/zouhir/eslint-config-standard-preact, we can reconsider, it adds a bunch of useful JSX rules too.